nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/CleanSurefireExecution.java

```diff
@@ -102,6 +102,34 @@ public void run() throws MojoExecutionException {
                                    + " " + domNode + " "
                                    + MojoExecutor.executionEnvironment(this.mavenProject, this.mavenSession,
                                                                        this.pluginManager));
+            Xpp3Dom domNodeArgLine = domNode.getChild("argLine");
```

Comment:
Can you add some comment for these changes? It's not clear why this is being done. Even better, you can extract this code into a method by itself, and then name it appropriately.

Reply:
thanks! done.